openedx/core/djangoapps/oauth_dispatch/docs/decisions/0007-include-organizations-in-tokens.rst

```diff
@@ -84,23 +86,36 @@ embedded in the scopes.
   and DOT Applications.
 
 * The new data model will also have a column for specifying organization
-  type: *content_provider* or *user_provider*. Initially, we will only
-  use *content_provider*.
+  type: *content_provider*, *user_provider*, *credit_provider*, etc.
+  Initially, we will only use *content_provider*.
 
-2. Organization Information in OAuth Tokens
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2. Organization and Users as Filters in OAuth Tokens
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The organization associated with the Application will be specified
+  in a new *filters* field in the JWT token.
```

Comment:
NIT: The ATM-machine-conscious pedant in me prefers `JWT` or `JSON Web Token` or `JSON Web Token (JWT)` here and elsewhere in this doc

Reply:
Duly noted.
I see this time both of you are on the same side of DRY.  :)